maple/bbsd.c

```diff
@@ -1531,8 +1533,17 @@ start_daemon(
 
     //sprintf(data, "%d\t%s\t%d\n", getpid(), buf, port);
     //f_cat(PID_FILE, data);
-    sprintf(data, "%d\n", getpid());
-    f_cat(PID_FILE, data);
+
+    if (port == -2)
+    {
+        sprintf(data, "%d\n", getpid());
+        f_cat(PID_FILE_UNIXSOCKET, data);
+    }
+    else
+    {
+        sprintf(data, "%d\n", getpid());
```

Comment:
This `sprintf` line is identical with the `sprintf` line above.
It will be better if they are merged.

Reply:
It looks promising now.